app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema.db")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connect.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connect.commit()
+
+    def all(self) -> list:
+        actors_cursor = self._connect.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.